common/src/domain/event/event.ts

```diff
@@ -0,0 +1,69 @@
+import { GREPA, LEPPA } from 'src/domain/berry';
+import { AddOneIngredient } from 'src/domain/event/event-util';
+import { Pokemon } from 'src/domain/pokemon';
+
+export interface Event {
+  name: string;
+  startDate: string;
```

Comment:
What do you think about Date vs string here? 

Date would enforce a correct format and it would mean we don't have to parse them all to date when listing current/future events in the frontend. 

Using string means we can list the dates in frontend without formatting a Date. 

If we go with Date we should be able to create an event like 
`new Date('2024-01-31')`

Reply:
Thanks, I much prefer the `Date` type.